pages/aid-centers/[slug].tsx

```diff
@@ -20,13 +20,29 @@ interface AidCenterInfoPageParams {
 }
 
 const AidCenterInfoPage: NextPage<AidCenterInfoPageProps> = (props) => {
-  const { name, note, assetsRequested, assetsUrgent, assetsFulfilled, callRequired } = props;
+  const { name, note, assetsRequested, assetsUrgent, assetsFulfilled, callRequired, slug, city, organization } = props;
+  const basePath = getBackendBaseUrl();
+
   return (
     <div className={styles.container}>
       <Head>
         <title>Gyűjtőközpont - {name}</title>
-        <meta name="description" content="" />
+
         <link rel="icon" href="/favicon.ico" />
+
+        <meta property="og:locale" content="hu_HU" />
+        <meta property="og:url" content={`${basePath}/aid-centers/${slug}`} />
+        <meta property="og:title" content={`${name} gyűjtőközpont szükségletei`} />
+        <meta
+          property="og:description"
+          content={`A ${city} településen működő ${organization.name} szervezetnek az alábbi adományokra van szüksége a legfrissebb adataink szerint:`}
```

Comment:
Ez akar mehet egy valtozoba/konstansba, mert ugyanaz az erteke a `description` meta-nak is.

Reply:
Pontosan. Csak azt még nem tudtam az előbb, hogy hogyan kell csinálni. Igaz most már a basePath kapcsán azt hiszem megtudtam. :)